src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -486,4 +511,25 @@ private function getSecuritySchemes(): array
 
         return $securitySchemes;
     }
+
+    private function appendSchemaDefinitions(\ArrayObject &$schemas, \ArrayObject $definitions): void
+    {
+        foreach ($definitions as $key => $value) {
+            $schemas[$key] = $value;
+        }
+    }
+
+    /**
+     * @var Model\Parameter[]
```

Comment:
Not needed?

Reply:
Yes or else I need to type `existingParameter` :)